src/components/Dashboard.vue

```diff
@@ -144,7 +144,6 @@ export default {
      * Creates a course on frontend with either user or API data
      */
     createCourse(course) {
-      // TODO: id?
       const randomId = Math.floor(Math.random() * Math.floor(1000));
```

Comment:
Can you actually leave a TODO here and make an item to look at it? This random ID thing seems very concerning to me

Reply:
O lmao yea I just thought we were using randomId for some reason.